de.fu_berlin.inf.dpp.ui.frontend/html/webpack.config.js

```diff
@@ -13,7 +13,7 @@ const createPage = (name) => {
   return new HtmlWebpackPlugin({
     filename: `${name}.html`,
     title: name,
-    template: 'template.ejs'
+    template: 'template.ejs',
```

Comment:
build configurations files dont have to be linted, our compiled GUI will run in browser but the config files will run in node.js environment

Reply:
These 'configuration files' are code. There should be a standard on how this JS code should look, and I think it should look exactly the same as all the other JS code. 
Linting is not something we are doing for the browsers. We use this to make our code more readable. This is why I would like to keep this included in the linting process.

Reply:
I just find it weird to lint both code and config which run in different environments with the same rules, anyway this is not an issue, you dont have to change it